src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -166,6 +166,10 @@ The `type` and default key is `"os"`.
 
 : _Optional_. A flag indicating whether the OS has been jailbroken or rooted.
 
+`theme`
+
+: _Optional_. Either `light` or `dark`. Describes whether the OS runs in dark mode or not.
```

Comment:
`theme` is under the `device` context, but the description is "whether the OS runs", we also have the `os` context, but its more related to the versioning, so it makes sense under the device I guess